docs/handbook/engineering/enterprise_edition.md

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
Please rename this to enterprise-edition.md. We are not using underscores in the documentation URL.